docs/challenges/language-support/csharp.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "C#"
+description: "Learn how to solve CodeCrafters challenges in C#"
+---
+
+# Local development
+
+To run the code for your challenge locally, you'll need `dotnet` installed. Our test runners use Dotnet 8.0 (as of April 2024).
+
+The script for your challenge (like `./your_sqlite3.sh`) will automatically compile your code using `dotnet` before executing it. It'll
+look something like this:
+
+```sh
+#!/bin/sh
+
+exec dotnet run --project . --configuration Release "$@"
```

Comment:
```suggestion
exec dotnet run --project . --configuration Release -- "$@"
```

(We're changing this here: https://linear.app/codecrafters/issue/CC-1170/c-spawn-scripts-are-not-properly-passing-arguments-to-main)